UnitTests/expected/Fonts-Dir-CustomName-Swift3.swift.out

```diff
@@ -30,13 +30,13 @@ extension Font {
 }
 
 struct CustomFamily {
-  enum SFNSDisplay: String, FontConvertible {
+  enum SfNsDisplay: String, FontConvertible {
```

Comment:
Yeah, I agree on this one. Don't have the time to fix it, but maybe this is solvable by changing the template? Or the new camelcase method?